libraries/networking/src/LimitedNodeList.cpp

```diff
@@ -92,7 +93,14 @@ LimitedNodeList::LimitedNodeList(unsigned short socketListenPort, unsigned short
     // check the local socket right now
     updateLocalSockAddr();
 
+    // TODO: Create a new thread, and move PacketReceiver to it
```

Comment:
I don't think we want this anymore right? As long as it's parented by the LNL it will move with the LNL